genomic_address_service/classes/reader.py

```diff
@@ -98,32 +81,14 @@ def read_matrix(self):
 
 
     def read_data(self):
-        ftype = self.guess_file_type(self.fpath)
-        dist_type = self.guess_dist_type(self.fpath, ftype, self.delim)
-        chunk = None
-        if ftype == 'text':
-            self.file_handle = open(self.fpath,'r')
-            self.header = next(self.file_handle).split(self.delim)
-        elif ftype == 'parquet':
-            self.file_handle = ParquetFile(self.fpath)
```

Comment:
Why parquet file type was removed? What was the logic behind it?

Reply:
It wasn't working. If you provided a validly formatted parquet file, GAS call wouldn't process it.

Here's the old code block:

https://github.com/phac-nml/genomic_address_service/blob/0d52d98146dba3b72bde4145f40639844d497dfc/genomic_address_service/classes/reader.py#L100-L123

In the second conditional block there's 2 cases handled:

`if ftype == 'text' and dist_type == 'pd':` - A pairwise distances text file.
`elif ftype == 'text' and dist_type == 'matrix':` - A matrix distances text file (which did not work for other reasons).

There was no case for something like:

`if ftype == 'parquet':`

So the code opened up Parquet files but then didn't do anything with them. I'm told it was left unfinished because of time constraints.